### ISSUE TITLE
`string-case` does not treat `forms` as `implicit progn`

### ISSUE THREAD

**Issue by VitoVan** [vindarel/cl-str#104]
case:

```lisp
CL-USER> (case 42
  (42
   (format t "this is 42~%")
   (format t "this is the answer~%")))
this is 42
this is the answer
NIL
```

string-case:

```lisp
CL-USER> (str:string-case "42"
  ("42"
   (format t "this is 42~%")
   (format t "this is the answer~%")))
this is 42
NIL
```

CLHS says the `forms` of `case` is an `implicit progn`.

Should cl-str follow this tradition?

**Comment by kilianmh** [comment]
This issue was fixed recently in #103 .
You can get the fixed version by either

1. updating the ultralisp dist
2. cloning the repository
3. wait for the next quicklisp release

**Comment by VitoVan** [comment]
Ah, great! Sorry for not searching thoroughly.
